src/pybind/mgr/nfs/tests/test_nfs.py

```diff
@@ -662,7 +662,6 @@ def _do_test_update_export(self, cluster_id, expected_exports):
         assert export.protocols == [4, 3]
         assert export.transports == ["TCP", "UDP"]
         assert export.fsal.name == "RGW"
-        assert export.fsal.user_id == "nfs.foo.bucket"
```

Comment:
Why are dropping this assert?

Reply:
the user_id is now the bucket owner, but the test mocks are returning empty results, so this ends up as an empty string in the tests.